config/environments/production.rb

```diff
@@ -55,7 +55,7 @@
   config.log_tags = [ :request_id ]
 
   # Use a different cache store in production.
-  # config.cache_store = :mem_cache_store
+  config.cache_store = :memory_store
```

Comment:
do you want to use the elasticache store?

Reply:
No, I'm explicitly looking for something that does 0 network calls.